test/test_torch.py

```diff
@@ -3305,6 +3306,44 @@ def test_from_buffer(self):
         self.assertEqual(floats.size(), 1)
         self.assertEqual(floats[0], 2.25)
 
+    def test_from_file(self):
+        size = 10000
+        filename = 'testPytorchStorageFromFile'
+        sys = platform.system()
+
+        def check(filename, share_mem):
+            s1 = torch.FloatStorage.from_file(filename, True, size, share_mem)
+            t1 = torch.FloatTensor(s1).copy_(torch.randn(size))
+            t1_copy = t1.new().resize_as_(t1).copy_(t1)
+            if share_mem and sys == 'Linux':
+                filename = os.path.join('/dev/shm', filename)
```

Comment:
Can't you just pass `share_mem` to `t2` as well? I wouldn't like to depend on these files being in `/dev/shm`